content/use-cases/list-xrp-in-your-exchange.md

```diff
@@ -37,22 +37,22 @@ From WebSocket and JSON-RPC API endpoints to the RippleAPI JavaScript library, f
 
 <span class="use-case-step-num">{{n.next()}}</span>
 <!-- <span class="use-case-step-length">(1 hour)</span> -->
-## [Get a sandbox XRP Ledger account](XRP Ledger Test Net.html)
+## [Get a sandbox XRP Ledger account](xrp-test-net-faucet.html)
 
 Use the XRP Ledger Test Net to get a sandbox account. Connect your `rippled` server to the Test Net to make test calls and get to know the XRP Ledger. Once you’re ready to transact in real XRP, you can switch over to transacting on the live XRP Ledger.
 
 
 <span class="use-case-step-num">{{n.next()}}</span>
 <!-- <span class="use-case-step-length">(1 hour)</span> -->
-## [Understand and code integrations to support the flow of funds](Listing XPR as an Exchange > Flow of Funds.html)
+## [Understand and code integrations to support the flow of funds](list-xrp-as-an-exchange.html#flow-of-funds)
 
 To support listing XRP, code integrations with the XRP Ledger to deposit XRP into your exchange, trade XRP on the exchange, rebalance XRP holding, and withdraw XRP from your exchange.
 
 
 ### Related Tasks
 
 - [Contribute Code to `rippled`](contribute-code-to-rippled.html)
-- [Subscribe to XRP Ledger Updates](x)
-- [Capacity Planning](x)
-- [Look Up an XRP Ledger Account’s Transaction History](x)
-- [Implement Destination Tags](x)
+- [Subscribe to XRP Ledger Updates](subscription-methods.html)
```

Comment:
Suggest changing the text of this link to be "Listen for new ledger versions"

The text as-is suggests signing up for the mailing list (which is also not a bad idea for a related task)